mod-source-record-manager-server/src/main/java/org/folio/services/mappers/processor/ProcessedSinglePlusConditionCheck.java

```diff
@@ -0,0 +1,19 @@
+package org.folio.services.mappers.processor;
+
+public class ProcessedSinglePlusConditionCheck extends ProcessedSingleItem {
```

Comment:
JavaDocs

Reply:
Thanks for your review, please keep in mind that code placed in org/folio/services/mappers/processor package is borrowed from mod-data-loader and will not be changed for now since it is a temporary solution.

Reply:
Thank goodness!